src/lib/elements/LimitedInput/LimitedInput.scss

```diff
@@ -0,0 +1,15 @@
+@import "vanilla-framework";
+
+.limited-input {
+  position: relative;
+
+  &__wrapper::before {
+    position: absolute;
+    pointer-events: none;
+    padding-left: $spv--small;
+    padding-bottom: calc(0.4rem - 1px);
+    padding-top: calc(0.4rem - 1px);
+    content: var(--immutable, "");
+    top: var(--top, "inherit");
```

Comment:
I'm aware that this isn't a perfect solution, but for now it's workable as advised by @petermakowski. I'll create a jira ticket to investigate this implementation and find something better, but for now this should unblock https://github.com/canonical/maas-ui/pull/5420